chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -48,18 +50,15 @@ pub struct PartialWitnessActor {
     /// Lock the value of mutable validator signer for the duration of a request to ensure consistency.
     /// Please note that the locked value should not be stored anywhere or passed through the thread boundary.
     my_signer: MutableValidatorSigner,
-    /// Epoch manager to get the set of chunk validators
```

Comment:
removal of comments here is an intended change: comments describing field usage get outdated quickly, also I don't see much value in those in the first place